aiobotocore/__init__.py

```diff
@@ -1 +1,3 @@
-__version__ = '2.13.3'
+import importlib.metadata
+
+__version__ = importlib.metadata.version("aiobotocore")
```

Comment:
There's a heated discussion over in PyPUG regarding whether `__version__` should even exist. It revealed that import-time is hurt if it's set like this.
So I'd suggest hiding it along with the import inside a module-level `__getattr__()` so that it's computed on access.

Reply:
In our case, we rely on `__version__` as part of session initialisation for creating the user-agent string. I believe the predictability of having `__version__` computed exactly once (at import time) outweighs the potentially shorter import time achieved when following the `__getattr__` approach. This is particularly true when session initialisation occurs in an async codepath (which it probably shouldn't) and the delayed import would block the event loop.

For reference, this is how `__getattr__` including caching could look like:

```python
def __getattr__(name):
    if name == "__version__":
        import importlib.metadata

        globals()["__version__"] = version = importlib.metadata.version(
            "aiobotocore"
        )

        return version

    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
```